velox/exec/tests/utils/OperatorTestBase.h

```diff
@@ -21,12 +21,13 @@
 #include "velox/core/Expressions.h"
 #include "velox/core/PlanNode.h"
 #include "velox/exec/tests/utils/QueryAssertions.h"
+#include "velox/exec/tests/utils/VectorTestBase.h"
 #include "velox/type/Variant.h"
 #include "velox/vector/FlatVector.h"
 #include "velox/vector/tests/VectorMaker.h"
 
 namespace facebook::velox::exec::test {
-class OperatorTestBase : public testing::Test {
+class OperatorTestBase : public testing::Test, public VectorTestBase {
```

Comment:
Nice. Can we re-use pool_ and vectorMaker_ variables from the base class (VectorTestBase) and remove these from OperatorTestBase?